packages/core/test/popover/popoverTests.tsx

```diff
@@ -594,6 +681,7 @@ describe("<Popover>", () => {
     interface IPopoverWrapper extends ReactWrapper<IPopoverProps, IPopoverState> {
         popover: HTMLElement;
         assertIsOpen(isOpen?: boolean): this;
+        assertOnInteractionCalled(called?: boolean): this;
         simulateTarget(eventName: string): this;
         findClass(className: string): ReactWrapper<React.HTMLAttributes<HTMLElement>, any>;
```

Comment:
here's that non-enzyme `findClass`

Reply:
This has to do with the other PR. I'll make sure I'm using the proper Enzyme approach in that one.